meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -2,3 +2,27 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 EXTRA_OENPM:witherspoon-tacoma = "-- --mode ibm"
 EXTRA_OENPM:p10bmc = "-- --mode ibm"
+
+inherit obmc-phosphor-systemd
+
+SRC_URI:append:p10bmc = " \
+	file://changeLogo.sh \
+	file://change-logo.service \
+	file://inspur-login-logo.svg \
+	file://inspur-logo-header.svg \
+    file://blankLogo.svg \
+	"
+
+SYSTEMD_SERVICE:${PN}:p10bmc = " change-logo.service"
+
+do_install:append:p10bmc() {
+    gzip ${WORKDIR}/inspur-login-logo.svg
```

Comment:
Curious: Is there a reason why the files aren't already zipped when checking them in? To be able to diff the SVG file if needed?

Reply:
Sorry, I just refer this link: https://github.com/ibm-openbmc/openbmc/commit/69ae2e079a8643ace8cf1d44988174cc8f2cf8a9
And I am not faimiler with webui, so you mean that we should zipped when checking them in?

Reply:
No, I was just curious as to why they weren't zipped in the first place. NM :)